salt/cloud/clouds/vmware.py

```diff
@@ -2546,7 +2548,11 @@ def create(vm_):
     # If it a template or if it does not need to be powered on then do not wait for the IP
     out = None
     if not template and power:
-        ip = _wait_for_ip(new_vm_ref, wait_for_ip_timeout)
+        try:
+            expected_ip = vm_['devices']['network']['Network adapter 1']['ip']
```

Comment:
We should iterate thought the list of network adapters and validate for each ips.